src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -57,90 +57,90 @@ internal class RazorSemanticTokensLegend
-            new SemanticTokenType("method name"),
-            new SemanticTokenType("module name"),
-            new SemanticTokenType("namespace name"),
-            new SemanticTokenType("operator - overloaded"),
-            new SemanticTokenType("parameter name"),
-            new SemanticTokenType("property name"), // 40
-            new SemanticTokenType("preprocessor keyword"),
-            new SemanticTokenType("preprocessor text"),
-            new SemanticTokenType("punctuation"),
-            new SemanticTokenType("record class name"),
-            new SemanticTokenType("record struct name"),
-            new SemanticTokenType("regex - alternation"),
-            new SemanticTokenType("regex - anchor"),
-            new SemanticTokenType("regex - character class"),
-            new SemanticTokenType("regex - comment"),
-            new SemanticTokenType("regex - grouping"), // 50
-            new SemanticTokenType("regex - other escape"),
-            new SemanticTokenType("regex - quantifier"),
-            new SemanticTokenType("regex - self escaped character"),
-            new SemanticTokenType("regex - text"),
-            new SemanticTokenType("string - escape character"),
-            new SemanticTokenType("struct name"),
-            new SemanticTokenType("text"),
-            new SemanticTokenType("type parameter name"),
-            new SemanticTokenType("string - verbatim"),
-            new SemanticTokenType("whitespace"), // 60
-            new SemanticTokenType("xml doc comment - attribute name"),
-            new SemanticTokenType("xml doc comment - attribute quotes"),
-            new SemanticTokenType("xml doc comment - attribute value"),
-            new SemanticTokenType("xml doc comment - cdata section"),
-            new SemanticTokenType("xml doc comment - comment"),
-            new SemanticTokenType("xml doc comment - delimiter"),
-            new SemanticTokenType("xml doc comment - entity reference"),
-            new SemanticTokenType("xml doc comment - name"),
-            new SemanticTokenType("xml doc comment - processing instruction"),
-            new SemanticTokenType("xml doc comment - text"), // 70
-            new SemanticTokenType("xml literal - attribute name"),
-            new SemanticTokenType("xml literal - attribute quotes"),
-            new SemanticTokenType("xml literal - attribute value"),
-            new SemanticTokenType("xml literal - cdata section"),
-            new SemanticTokenType("xml literal - comment"),
-            new SemanticTokenType("xml literal - delimiter"),
-            new SemanticTokenType("xml literal - embedded expression"),
-            new SemanticTokenType("xml literal - entity reference"),
-            new SemanticTokenType("xml literal - name"),
-            new SemanticTokenType("xml literal - processing instruction"), // 80
-            new SemanticTokenType("xml literal - text"),
+            "namespace", // 0
```

Comment:
lol, one of us is going to have definite merge conflicts here 😄

Reply:
Normally I would volunteer as tribute but IIRC you're PR is gonna be delayed a bit so I'ma go first. I don't think the conflicts should be too bad, you'll just have to take what already exists in your PR and remove the `new SemanticTokenType` stuff.